loky/backend/reduction.py

```diff
@@ -48,17 +43,17 @@ def h(cls):
         pass
 
 
-register(type(_C().f), _reduce_method)
-register(type(_C.h), _reduce_method)
+dispatch_table[type(_C().f)] = _reduce_method
+dispatch_table[type(_C.h)] = _reduce_method
```

Comment:
I prefer to keep the `register` public API to keep a the public API of this module compatible with the API of the `reduction` module of `multiprocessing`.

Reply:
Maybe then modify `register` to allow use as decorator?